backend/src/main/kotlin/app/ehrenamtskarte/backend/graphql/shared/exceptions/ApplicationDataIncompleteException.kt

```diff
@@ -0,0 +1,7 @@
+package app.ehrenamtskarte.backend.graphql.shared.exceptions
```

Comment:
I would put exceptions direct in graphql package (just because they are "self-sufficient" enough), but this might be too personal